modules/update-mirror-repository.adoc

```diff
@@ -114,6 +114,17 @@ $ oc image mirror  -a ${LOCAL_SECRET_JSON} --from-dir=${REMOVABLE_MEDIA_PATH}/mi
 +
 <1> For `REMOVABLE_MEDIA_PATH`, you must use the same path that you specified when you mirrored the images.
 
+... Use `oc` command-line interface (CLI) to log in to the cluster that you are upgrading.
```

Comment:
More of a note on this entire section:

Shouldn't it be 4. a. i.? Currently, the second set of lists is unordered, but there are two items there.

Reply:
It uses a bulleted list instead of numbered steps because it is one of two options.